app/views/catalog/_index_media_object.html.erb

```diff
@@ -21,7 +21,7 @@ Unless required by applicable law or agreed to in writing, software distributed
   <% index_fields(document).each do |field_name, field| -%>
     <% if should_render_index_field? document, field %>
 	    <dt class="blacklight-<%= field_name.parameterize %>"><%= render_index_field_label document, field: field_name %></dt>
-	    <dd class="blacklight-<%= field_name.parameterize %>"><%= doc_presenter.field_value field_name %></dd>
+	    <dd class="blacklight-<%= field_name.parameterize %>"><%= doc_presenter.field_value(doc_presenter.configuration.index_fields.fetch(field_name)) %></dd>
```

Comment:
Address: Blacklight 7 deprecation log message

Reply:
Is the deprecation about being a string when a symbol is required?  Could do `field_name.to_sym` instead?  But maybe I'm over simplifying the problem.

Reply:
I haven't gone far into the Blacklight codebase. The deprecation was on field_value's acceptance of a string argument and in Blacklight 7, field_value wants an instance of type StringBlacklight::Configuration::Field. I'll check the `field_name.to_sym`.

Reply:
Both `string` and `symbols` are deprecated with `field_value`.